examples/aiohttp_example.py

```diff
@@ -0,0 +1,104 @@
+import json
+import logging
+import os
+# from asyncio import sleep as async_sleep
+from secrets import token_hex
+from datetime import datetime
+from aiohttp import web
+from peewee import Model, CharField, TextField, DateTimeField
+from peewee_async import PooledPostgresqlDatabase, Manager
+
+logger = logging.getLogger(__name__)
+
+database = PooledPostgresqlDatabase(
+    'examples', user='postgres', password='postgres',
+    host='db-postgres', port=5432,
+    min_connections=10, max_connections=100,
+)
+
+objects = Manager(database)
```

Comment:
Are you sure you want to use Manager and api we are going to remove it future version?

Reply:
@kalombos I think we should have example for that and keep it explicitly unitl some version. I know projects that are using peewee-async and we should not break it. But I agree we'd need to communicate that properly.

Let's leave this example as-is and for future examples and documentation we'll mention that this API will be frozen in some version like `0.99` and removed in `1.0` or something.

Reply:
Oh, i got your idea. Let's leave the example no problem :)